reservations/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+from core.models import TimeStampModel
+
+class Reservation(TimeStampModel):
+    check_in  = models.DateField()
+    check_out = models.DateField()
+    nights    = models.IntegerField()
```

Comment:
`nigths` 컬럼에 어떤 값이 저장되죠?
```suggestion
    nights    = models.IntegerField()
```

Reply:
nights 에는 게스트가 예약한 일 수를 담는 필드입니다. 모델링 리뷰 시간에 병민님께서 굳이 없어도 될것 같다고 하셔서 방금 지우고 다시 push 했습니다 !